charts/posit-chronicle/README.md

```diff
@@ -43,74 +43,99 @@ helm search repo rstudio/posit-chronicle -l
-        mountPath: "/var/lib/rstudio-server/audit"
-      env:
+initContainers:
+  - name: chronicle-agent
+    restartPolicy: Always
+    image: ghcr.io/rstudio/chronicle-agent:2025.03.0
+    env:
       - name: CHRONICLE_SERVER_ADDRESS
-        value: "http://chronicle-server.default"
+        value: "http://<service>.<namespace>"
 ```
 
 And here is an example of Helm values for Connect, where a **Connect**
 API key from a Kubernetes Secret is used to unlock more detailed metrics:
 
 ```yaml
-pod:
-  sidecar:
-    - name: chronicle-agent
-      image: ghcr.io/rstudio/chronicle-agent:2025.03.0
-      env:
-      - name: CHRONICLE_SERVER_ADDRESS
-        value: "http://chronicle-server.default"
-      - name: CONNECT_API_KEY
-        valueFrom:
-          secretKeyRef:
-            name: connect
-            key: apikey
+initContainers:
+- name: chronicle-agent
+  restartPolicy: Always
+  image: ghcr.io/rstudio/chronicle-agent:2025.03.0
+  env:
+    - name: CHRONICLE_SERVER_ADDRESS
+      value: "http://<service>.<namespace>"
+    - name: CHRONICLE_CONNECT_APIKEY
+      valueFrom:
+        secretKeyRef:
+          name: connect
+          key: apikey
 ```
 
-Note that it is up to the user to provision this Kubernetes Secret for the
-Connect API key.
+It is up to the user to provision this Kubernetes Secret for the
+Connect API key. The `extraObjects` value in the Connect chart can be used to
+create the secret and mount it to the Chronicle agent container. Due to the
+nature of the Chronicle agent, the pod may need to be restarted to pick up
+changes to the secret after initial deployment.
+
+## HTTPS Configuration
+
+Chronicle can be configured to use HTTPS for secure communication. The
+`config.HTTPS` section of the configuration allows you to specify the certificate
+and key files to use for HTTPS. Both `config.HTTPS.Certificate` and
+`config.HTTPS.Key` are expected to be paths to files accessible by Chronicle.
+The `extraSecretMounts` value can be used to mount the certificate and key files
+into the Chronicle pod. Here is an example of how to do this, assuming that
+the certificate and key files are stored together in a Kubernetes TLS secret:
+
+```yaml
+extraSecretMounts:
+  - name: chronicle-https
+    mountPath: /etc/chronicle/ssl
+    secretName: chronicle-https
+    items:
+      - key: tls.crt
+      - key: tls.key
+config:
+  HTTPS:
+    Enabled: true
+    Certificate: "/etc/chronicle/ssl/tls.crt"
+    Key: "/etc/chronicle/ssl/tls.key"
+```
 
 ## Storage Configuration
 
-Chronicle can be configured to persist data to a local Kubernetes Volume, AWS S3, or both.
+Chronicle can be configured to persist data to local storage, AWS S3, or both.
 
-The default configuration uses a local volume, which is suitable if you'd like to
-access and analyze the data within your cluster:
+The default configuration uses a local volume with persistence enabled, which
+is suitable if you'd like to access and analyze the data within your cluster:
```

Comment:
Based on the discussions we had, we want to default to the S3 backend in the chart.